testsuite/regressiontests/issue-1085.sli

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
should this be part of this PR?

Reply:
Yes. @stinebuu suggested as both this PR and #1093 fixes #1085, #1093 should be closed in favor of this. Therefore this PR got the regressiontest.

Reply:
ok